internal/compiler-interface/src/main/contraband/incremental.json

```diff
@@ -124,29 +117,6 @@
           ]
         },
         {
-          "name": "antStyle",
```

Comment:
Ant style was already deprecated and removed from the sources. This commit only removes the rest that was not removed back then. I don't know who did it, but there was no implementation making use of this option, IIRC. In any case, I think that now it's the time to remove it because this option was never used and we kept it in the 0.13.13 cycle for reproducibility.